src/app/BUILD.gn

```diff
@@ -189,6 +189,9 @@ static_library("interaction-model") {
     "reporting/ReportSchedulerImpl.h",
     "reporting/SynchronizedReportSchedulerImpl.cpp",
     "reporting/SynchronizedReportSchedulerImpl.h",
+
+    # TODO: Reporting changes data model versions and is implemented in data_model.
```

Comment:
This comment really confused me.  This is talking about cluster DataVersions, not the (compile-time-constant) data model revision, right?

Please rephrase to make it clearer.